fab/services/templates/celery_bash_runner.sh

```diff
@@ -15,7 +15,27 @@
 # the Celery process to do a warm shutdown and we are
 # free to start another bash process under supervisor.
 trap 'kill -TERM $PID' TERM INT
+
+HOSTNAME=""
+ARGS=""
+for i in "$@"
```

Comment:
i hate this loop, but there didn't seem to be a great way to do this. i feel like it's probably OK for this isolated scenario